packager/pom.xml

```diff
@@ -59,7 +59,7 @@
     <bouncycastle.version>1.46</bouncycastle.version>
     <commons.cli.version>1.2</commons.cli.version>
     <gson.version>2.2.4</gson.version>
-    <guava.version>19.0</guava.version>
+    <guava.version>30.0-android</guava.version>
```

Comment:
just curious about this android dependency :)

Reply:
https://mvnrepository.com/artifact/com.google.guava/guava after the 23.0, every dependency is either `-jre` or `-android`
I used the `-android` version to match the one used by the google cloud storage dependency